### PR TITLE
Updated depecration link of radiospectra

### PR DESCRIPTION
**Pull request by dpshelio**
This is linked with sunpy/spectra#20

I'm not sure whether this should be backported. Probably github redirect links of old repos that change names for a bit...

### CODE REVIEW

**Comment by Cadair** [comment]
Once the documentation is alive would it not be better to refer to that?

**Comment by dpshelio** [comment]
Sure! what we need to do to put it under `docs.sunpy.org/projects`?

**Comment by Cadair** [comment]
@dpshelio ask and ye shall receive: http://docs.sunpy.org/projects/radiospectra/en/latest/

**Comment by nabobalis** [comment]
Looks fine, just add docs link and we can merge this.

File: docs/guide/data_types/spectra.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Spectra
 =======
 
 .. warning:: This module is under development! It is being moved into its own
-             `repository <https://github.com/sunpy/spectra>`__.
+             `repository <http://docs.sunpy.org/projects/>`__.
```
Comment (Cadair) [inline docs/guide/data_types/spectra.rst]: this URL is wrong.

Comment (nabobalis) [reply docs/guide/data_types/spectra.rst]: HOW ON EARTH DID I SCREW THAT ONE UP

**Comment by Cadair** [comment]
let's make sure the docs build

**Pull request merged**